operations/operations/publishing/publishing-guidelines/documentation-style-guide.md

```diff
@@ -86,6 +86,27 @@ Use active voice in preference to passive voice. Active voice has the subject of
 
 Use the present tense.
   - For example: "Sharing this link lets other users view the linked message." as opposed to "Sharing this link will let other users view the linked message."
+  
+## Terminology Guide
+
+Throughout Mattermost, you’ll see the below words, terms are mentioned and used. Their definitions and reason for usage are defined below.
+
+| Terms          | Description        | Usage             |
+| ------------- |-------------| --------------------|
+| Mattermost | Used to refer to [Mattermost](https://mattermost.com/about-us/) the company and [Mattermost](https://mattermost.com/product/) the product. | <ul><li> Mattermost is an open-source, self-hostable online chat service with file sharing, search, and integrations</li> <li> Configure Mattermost on your server</li></ul> |
+| installation | Refers to an on-premises Mattermost environment comprised of hardware (server/s), database, filestore, etc., which hosts the required files to run Mattermost. | <ul><li> Your Mattermost installation includes physical servers </li></ul> |
+| deployment | Used to refer to making a new installation widely accessible. It can also refer to releasing an updated version, patch, etc. | <ul><li> Desktop App Deployment Guide </li></ul> |
```

Comment:
```suggestion
| deployment | Used to refer to making a new installation widely accessible. It can also refer to releasing an updated version, patch, etc. | <ul><li> Desktop App deployment guide </li></ul> |
```